package.json

```diff
@@ -5,11 +5,13 @@
   "main": "index.js",
   "type": "module",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
     "start": "nodemon index.js",
     "task1": "nodemon module_3/task1.js",
     "task2": "nodemon module_3/task2.js",
-    "task3": "nodemon module_3/task3.js"
+    "task3": "nodemon module_3/task3.js",
+    "test": "jest src/test/unit src/test/integration",
+    "test:coverage": "jest src/test/unit src/test/integration --coverage",
```

Comment:
The coverage command is not working as expected, it should display information about files covered and it is showing 0% in all
![image](https://github.com/juasmartinezbelS4N/nodejs_course/assets/12193300/2e2ab4b0-578b-462d-a747-e87b0b6d3a9e)

Reply:
This is very weird, I just tested it in the work computer and my computer and I got both working

<img width="812" alt="Captura de pantalla 2024-03-18 a las 10 05 14 a  m" src="https://github.com/juasmartinezbelS4N/nodejs_course/assets/56047417/41ffff13-e28d-4e64-84a4-528092787101">

![432594245_1325987031412586_750135584427623995_n](https://github.com/juasmartinezbelS4N/nodejs_course/assets/56047417/7d023a3b-24eb-48da-8917-8cd25a5ac821)

How are you running the tests?, or do you notice something that I might be missing?

Reply:
let me check, it is very weird